initial setup for floydhub instructions (#1)
whatrocks/pytorch.github.io: _get_started/installation/floydhub.md

Once you are logged in, you will be brought to your [FloydHub web dashboard](https://www.floydhub.com/){:target="_blank"} where you will create your first project.  You can even learn more about FloydHub through a set of [simple video tutorials](https://youtu.be/FcsjqQ2QdLQ){:target="_blank"}.

### FloydHub Environments

FloydHub provides no setup required, pre-configured environment to help you build your deep learning projects. [FloydHub Deep Learning Environments](https://docs.floydhub.com/guides/environments/){:target="_blank"} are a set of Ubuntu-based virtual machines that allow you to build and run [PyTorch machine learning based applications](https://docs.floydhub.com/guides/pytorch/).

## FloydHub Workspace: GPU-based Virtual Machines
{: #floydhub-workspace}

> Generally, you will be using [FloydHub Workspace](https://www.floydhub.com/product/build){:target="_blank"} to begin with PyTorch. FlodyHub Workspace is an interactive environment (Jupyter Lab) for developing and running code. You can run Jupyter notebooks, Python scripts and much more. All the files and data in your workspace will be preserved for you, across restarts. You can think of it as your persistent, on-demand machine on the cloud.

Reviewer: ```suggestion
Generally, you will be using [FloydHub Workspace](https://www.floydhub.com/product/build){:target="_blank"} to begin with PyTorch. FlodyHub Workspace is an interactive environment (Jupyter Lab) for developing and running code. You can run Jupyter notebooks, Python scripts and much more. All the files and data in your workspace will be preserved after shutdown.
```
Removed some sentences that look like marketing terms.
